### PR TITLE
feat(CC): add datasource CC central networks connections

### PR DESCRIPTION
**Pull request by luoping-12345**
**What this PR does / why we need it**:
add datasource CC central networks connections

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```
add datasource CC central networks connections
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cc" TESTARGS="-run TestAccDataSourceCcCentralNetworkConnections_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cc -v -run TestAccDataSourceCcCentralNetworkConnections_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceCcCentralNetworkConnections_basic
=== PAUSE TestAccDataSourceCcCentralNetworkConnections_basic
=== CONT  TestAccDataSourceCcCentralNetworkConnections_basic
--- PASS: TestAccDataSourceCcCentralNetworkConnections_basic (318.84s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cc        318.891s
```

### CODE REVIEW

File: docs/data-sources/cc_central_network_connections.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<a name="central_network_connections_connection_point_pair_struct"></a>
+The `connection_point_pair` block supports:
+
+* `id` - The point ID of a central network connection.
+
+* `project_id` - The point project ID of a central network connection.
+
+* `region_id` - The point region ID of central network connection.
+
+* `site_code` - The site code of the point  of central network connection.
```
Comment (houpeng80) [inline docs/data-sources/cc_central_network_connections.md]: remove the extra spaces

File: huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_connections_test.go [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+  is_cross_region    = "false"
+}
+  
+locals {
+  connsById = data.huaweicloud_cc_central_network_connections.filter_by_id.central_network_connections
+  connsByState = data.huaweicloud_cc_central_network_connections.filter_by_state.central_network_connections
+  connsByBwType = data.huaweicloud_cc_central_network_connections.filter_by_bwtype.central_network_connections
+  connsByType = data.huaweicloud_cc_central_network_connections.filter_by_type.central_network_connections
+  connsByCross = data.huaweicloud_cc_central_network_connections.filter_by_cross.central_network_connections
+  connsByNotCross = data.huaweicloud_cc_central_network_connections.filter_by_not_cross.central_network_connections
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/cc/data_source_huaweicloud_cc_central_network_connections_test.go]: align the equal signs

File: docs/data-sources/cc_central_network_connections.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+Use this data source to get the list of CC central network connection.
+
+## Example Usage
+
+```hcl
+variable "central_network_connection_id" {}
+
+resource "huaweicloud_cc_central_networks" "test" {
+  connection_id = var.central_network_connection_id
```
Comment (houpeng80) [inline docs/data-sources/cc_central_network_connections.md]: central_network_id is required,  should be added

File: docs/data-sources/cc_central_network_connections.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
+---
+subcategory: "Cloud Connect (CC)"
+---
+
+# huaweicloud_cc_central_network_connections
+
+Use this data source to get the list of CC central network connection.
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cc_central_network_connections.md]:
```suggestion
Use this data source to get the list of CC central network connections.
```

File: docs/data-sources/cc_central_network_connections.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+```hcl
+variable "central_network_id" {}
+variable "central_network_connection_id" {}
+
+resource "huaweicloud_cc_central_networks" "test" {
+  central_network_id = var.central_network_id
+  connection_id      = var.central_network_connection_id
+}
+```
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cc_central_network_connections.md]: this example is not correct

File: docs/data-sources/cc_central_network_connections.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the resource.
+  If omitted, the provider-level region will be used.
+
+* `central_network_id` - (Required, String) Specifies the central network ID.
+
+* `connection_id` - (Optional, String) Specifies the central network connection ID.
+
+* `state` - (Optional, String) Specifies the central network connection status.
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cc_central_network_connections.md]:
```suggestion
* `status` - (Optional, String) Specifies the central network connection status.
```

File: docs/data-sources/cc_central_network_connections.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+* `name` - The central network connection name.
+
+* `description` - The central network connection description.
+
+* `created_at` - The creation time. The time is in the **yyyy-MM-ddTHH:mm:ss** format.
+
+* `updated_at` - The update time. The time must be in the **yyyy-MM-ddTHH:mm:ss** format.
+
+* `state` - The central network connection status.
```
Comment (Jason-Zhang9309) [inline docs/data-sources/cc_central_network_connections.md:69]: status

Comment (Jason-Zhang9309) [inline docs/data-sources/cc_central_network_connections.md]:
```suggestion
* `status` - The central network connection status.
```

**Pull request merged**